### PR TITLE
fix(css): Fix print view in viewer and single-file share

### PR DESCRIPTION
**Pull request by mejo-**
### 📝 Summary

Also adds screenshot comparison tests for print view in viewer and single-file share.

* Resolves: #4536

### 🏁 Checklist

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits
- [x] [Tests](https://github.com/nextcloud/text#-testing-the-app) (unit, integration and/or end-to-end) passing and the changes are covered with tests
- [x] Documentation is not required

### CODE REVIEW

File: cypress/snapshots/base/print.spec.js/print view in viewer.png
Comment (juliushaertl) [inline cypress/snapshots/base/print.spec.js/print view in viewer.png:1]: Haven't tested yet or looked in detail but that snapshot looks wrong to me ;)

Comment (mejo-) [reply cypress/snapshots/base/print.spec.js/print view in viewer.png:1]: Hehe indeed, good catch. That's fixed now.

**Comment by mejo-** [comment]
The snapshot comparison will only succeed after the JS assets got updated. So let's try to compile+push them into the PR.

**Review by juliushaertl — APPROVED** [review]
Nice, especially having those tests 👍

**Comment by mejo-** [comment]
> Nah, looks like the font stack used by your local snapshots is different from CI 🙈

Yeah, the joy of visual comparison tests :zany_face: I updated it now with the snapshots created by CI.

**Comment by mejo-** [comment]
/backport 90ce11e5cdba28c123bc79fad9fe5d4a462bcbfe,aff3a1add26cf0edc12f2c624c4a109edf320a46,280948b35e63b46cbf446657c73492343bf4f79d,035f5cc900528277b3ead8e8640e885cb80af949,a73f1f339f78aed55c7f2efc28e384ff3e39a641,9af378da3ba7675db4b3f2f9244c8b4e1d923bd0,624651e86f7f681f51fa8e35931f4ab9a6f33a87,5c7d3918dbc790273aa6a9dacf2c36f701b7d7ea to stable28

**Comment by suther** [comment]
Is this issue at least solved or still in progress. 
Currently it look like this at my PC if I try to print any note in nextcloude notes:

image

Can I give helping hand in any way to solve this. 

BTW: On Android there are no print-functionality at all for the notes. A huge downside in my opinion for a note -app

**Pull request merged**